File: Java-Kotlin/src/main/java/array/_1470.java
package array;

public class _1470 {

    public int[] shuffle(int[] nums, int n) {
        int[] result = new int[2 * n];
        for (int i = 0; i < n; i++) {
            int cur = 2 * i;
            int next = cur + 1;
            result[cur] = nums[i];
            result[next] = nums[i + n];
        }
        return result;
    }
}

File: Go-Solutions/1470.go
package main

func shuffle(nums []int, n int) []int {
	ret := make([]int, 2*n)
	for i := 0; i < n; i++ {
		cur := 2 * i
		next := cur + 1
		ret[cur] = nums[i]
		ret[next] = nums[i+n]
	}
	return ret
}
